PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrack3DMultKtExtended.cxx

```diff
@@ -647,38 +647,75 @@ struct femtoUniversePairTaskTrackTrack3DMultKtExtended {
         }
       }
 
+      TRandom2* randgen = new TRandom2(0);
```

Comment:
Allocations/de-allocations of memory should not be done at pair level

Reply:
Hi Victor, thanks for pointing this out. In the updated code, I have allocated the memory before pair level and de-allocated the memory in the process functions. Please have a look at it.